sql/updates/world/master/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,287 @@
+ALTER TABLE `areatrigger`
+    ADD COLUMN `spawnDifficulties` VARCHAR(100) NOT NULL DEFAULT '0' COLLATE 'utf8mb4_unicode_ci' AFTER `MapId`;
```

Comment:
UpperCamelCase is current standard for new fields

Reply:
Done, it was already UCC in c++, fixed the SQL